source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -4135,16 +4136,19 @@ void Graph::drawGraph(ImVec2 mousePos)
             ed::NavigateToContent(0.0f);
         }
```

Comment:
from my local experiments, I believe this small "centering" block could move above so you can get rid of the `shouldProcessInput` variable and wrap everything under the same `if`

Reply:
Good catch! Changes pushed.